src/vcpkg/base/downloads.cpp

```diff
@@ -23,16 +24,16 @@ using namespace vcpkg;
 
 namespace
 {
-    constexpr StringLiteral vcpkg_curl_user_agent_header =
-        "User-Agent: vcpkg/" VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING " (curl)";
+    constexpr StringLiteral vcpkg_curl_user_agent =
+        "vcpkg/" VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING " (curl)";
 
-    void add_curl_headers(Command& cmd, View<std::string> headers)
+    void set_common_curl_options(CURL* handle, const char* url, curl_slist* request_headers)
```

Comment:
Should these things be marked `curl_easy` since I believe that `CURL*` needs to be an easy handle?